doc/7/protocols/websocket/introduction/index.md

```diff
@@ -23,3 +23,23 @@ This application-level PING has been especially added for web browsers, which do
 When run in a browser, our Javascript SDK uses that feature for its keep-alive mechanism: a message will periodically be sent to Kuzzle in the form `"{"p":1}"` through websocket.
 That message will call a response from Kuzzle in the form `"{"p":2}"` for the SDK to keep the connection alive.
 
+### Cookie Authentication
+
+Kuzzle does support cookie authentication, meaning that when you're making a request, you can instruct Kuzzle to store the token securely inside a cookie in your browser, instead of returning it in the response.
```

Comment:
Can we have the schema here for more clarity?

Reply:
I will just update the schema before adding it

Reply:
NVM, we might want to do that later